wai-videos/video-introduction/video-introduction.id.vtt

```diff
@@ -97,19 +97,19 @@ dan diadopsi sebagai standar di Eropa dengan nama EN 301 549.
 WCAG dibangun berdasarkan empat prinsip utama:
 
 00:02:33.560 --> 00:02:39.920 align:middle
-Pertama, Dapat Dipersepsikan - sebagai contoh, seseorang bisa melihat sebuah konten, atau mendegarnya.
+Pertama, dapat diterima indra - sebagai contoh, seseorang bisa melihat sebuah konten, atau mendegarnya.
 
 00:02:40.880 --> 00:02:47.160 align:middle
 Dapat Dioperasikan - sebagai contoh, seseorang bisa menggunakan komputer dengan mengetik, atau melalui suara.
 
 00:02:48.000 --> 00:02:53.360 align:middle
-Dapat Dipahami - sebagai contoh, seseorang menerima dengan bahasa yang jelas dan sederhana.
+Dapat Dimengerti - sebagai contoh, seseorang menerima dengan bahasa yang jelas dan sederhana.
```

Comment:
@frirasyidi Because this translates a key WCAG principle, I leave it for discussion with other reviewers.

Would you explain the rationale behind your proposal?